docs/requirements.txt

```diff
@@ -2,3 +2,8 @@ pyenchant
 Sphinx>=4.5.0
 sphinxcontrib-spelling
 blacken-docs
+
+# Due to Sphinx's viewcode extension, the docs are dependent on Django itself.
+# Below we install Django from the local directory. This assumes installation
```

Comment:
I'm not sure if it's mentioned in the style guide, but try to write comments without "we".  In this case, I would write, "The period below installs Django..."

Reply:
(this is mentioned in the style guide, sorry I missed it before :+1:)